src/main/scala/firrtl/ir/Serializer.scala

```diff
@@ -302,7 +302,11 @@ object Serializer {
     // WIR
     case firrtl.CDefMemory(info, name, tpe, size, seq, readUnderWrite) =>
       if (seq) b ++= "smem " else b ++= "cmem "
-      b ++= name; b ++= " : "; s(tpe); b ++= " ["; b ++= size.toString(); b += ']'; s(info)
+      b ++= name; b ++= " : "; s(tpe); b ++= " ["; b ++= size.toString(); b += ']'
+      if (readUnderWrite != ReadUnderWrite.Undefined) {
+        b += ' '; b ++= readUnderWrite.toString
+      }
```

Comment:
It would be better to unconditionally serializing the read-under-write behavior than to have an implicit default. This only affects the MFC in terms of making this a breaking change.

Reply:
I can change it. The idea behind not serializing the default was that it would maintain backwards compatibility for most circuits (since RUW is rarely set).

Reply:
changed